packages/main/src/Table.ts

```diff
@@ -488,6 +472,8 @@ class Table extends UI5Element {
 		return {
 			table: {
 				"grid-template-columns": this._gridTemplateColumns,
+				"height": this.height,
```

Comment:
as mentioned above I think we should avoid such dimension properties, we do that for the column but it is not only for column but all cells inside.

Reply:
Ack

Reply:
Maybe we need something similar to the GridTable here? Where the height is calculated by the Table itself based on the number of items? Let's discuss offline